Sources/ODP/OdpVuidManager.swift

```diff
@@ -17,14 +17,17 @@
 import Foundation
 
 class OdpVuidManager {
-    var vuid: String = ""
+    private var _vuid: String = ""
+    private(set) var enabled: Bool
     let logger = OPTLoggerFactory.getLogger()
-
-    // a single vuid should be shared for all SDK instances
-    static let shared = OdpVuidManager()
```

Comment:
We should keep it a singleton, so can share a single vuid per device.

Reply:
Declared singleton and add initialize method.
```
static let shared = VuidManager()
    
    func initialize(enabled: Bool) {
        self.enabled = enabled
        if enabled {
            self._vuid = load()
        } else {
            self.remove()
        }
    }
    
```